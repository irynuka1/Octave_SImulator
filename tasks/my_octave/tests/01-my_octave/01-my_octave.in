L 2 6
7 7 -1 -1 -10 -6
-9 2 2 7 6 2
P 1
D 0
P 0
L 5 2
1 -9
10 -7
-4 9
-4 2
-2 8
M 1 0
P 2
F 1
Q