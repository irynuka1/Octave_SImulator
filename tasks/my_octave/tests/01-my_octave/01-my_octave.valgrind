==12478== Memcheck, a memory error detector
==12478== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==12478== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==12478== Command: /home/irynuka1/Desktop/Programs/Tema2_1/check_my_octave_v1.1/check_my_octave/my_octave
==12478== Parent PID: 12477
==12478== 
==12478== 
==12478== HEAP SUMMARY:
==12478==     in use at exit: 0 bytes in 0 blocks
==12478==   total heap usage: 26 allocs, 26 frees, 8,632 bytes allocated
==12478== 
==12478== All heap blocks were freed -- no leaks are possible
==12478== 
==12478== For lists of detected and suppressed errors, rerun with: -s
==12478== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
