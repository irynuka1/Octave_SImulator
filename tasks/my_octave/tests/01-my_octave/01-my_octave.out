No matrix with the given index
2 6
7 7 -1 -1 -10 -6 
-9 2 2 7 6 2 
88 9996 9988 9943 9943 9983 
133 56 9983 9948 9865 9933 
9898 9997 22 67 94 42 
9961 9983 8 18 52 28 
9921 2 18 58 68 28 
